pkg/prompts/prompts.go

```diff
@@ -917,27 +963,27 @@ func PromptChain(
 	prompter Prompter,
 	prompt string,
 	subnetNames []string,
-	includePChain bool,
-	includeXChain bool,
-	includeCChain bool,
-	avoidBlockchainName string,
+	avoidPChain bool,
```

Comment:
why changing the logic to all avoids?

Reply:
we double careful this may broke some calls to this func

Reply:
probably used on key transfer. should check

Reply:
my bad. fixed